ci/builders/linux_host_engine.json

```diff
@@ -210,6 +210,40 @@
                     "type": "local"
                 }
             ]
+        },
+        {
+            "archives": [
+                {
+                    "name": "wasm_release",
+                    "base_path": "out/wasm_release/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                        "out/wasm_release/zip_archives/flutter-web-sdk.zip",
+                    ]
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_custom_vars": {
+                "download_emsdk": true,
+                "download_android_deps": false
+            },
+            "gn": [
+                "--runtime-mode",
+                "release",
+                "--web",
+                "--build-canvaskit"
+            ],
+            "name": "wasm_release",
+            "ninja": {
+                "config": "wasm_release",
+                "targets": [
+                    "flutter/web_sdk",
```

Comment:
I believe this comma is causing the build failure, can you please validate the json file running `python -mjson.tool filename`?

Reply:
Done.